examples/inferencing/running-on-gke.md

```diff
@@ -0,0 +1,185 @@
+## Deployment
+
+The deployment to GKE is currently detached from the training pipeline. The model artifact that is a result of training will need to be manually deployed to the cluster using these instructions.
+
+
+### Create an Environment file
+
+&lt;TODO> / This section will need to change after CPT-0.7.0 release \
+ \
+An environment variable file containing the configuration for the GKE cluster and the model needs to be created. The full specification for the cluster configuration can be found [here](https://github.com/GoogleCloudPlatform/ai-infra-cluster-provisioning#configuration-for-users). A sample configuration is available in the repository at [llm-pipeline-examples/src/gke/sample_environment_config.list](https://github.com/GoogleCloudPlatform/llm-pipeline-examples/blob/main/src/gke/cluster_config.list)
```

Comment:
Please give a short description on what the sample config do

Reply:
There is a description field already for each of the entries in the table, do you want a more detailed description for any of them?

Reply:
I am referring to the specific config in sample_environment_config.list . We can say something like "If we use this sample environment config, it will create a GKE cluster for you that has 1 node, 2 GPUs, etc.... "